userbenchmark/torch_trt/run.py

```diff
@@ -8,14 +8,20 @@
 import os
 import time
 from datetime import datetime
-from typing import List
+from typing import List, Union
 
+from torchbenchmark.util.experiment.instantiator import (
+    TorchBenchModelConfig,
+    load_model_isolated,
+)
 from torchbenchmark import (
+    ModelTask,
     load_canary_model_by_name,
     load_model_by_name,
-    list_models,
+    _list_model_paths,
```

Comment:
It is better to use the `list_models()` API from `torchbenchmark.util.experiment.instantiator`. functions starting with `_` are not supposed to be used by userbenchmarks.

Reply:
@gs-olive can you help address this so that we can merge this PR?

Reply:
Thank you for the review, and apologies for the delay on this. I will raise another PR to address this issue and clean it up, if that would be preferable? Additionally, I was wondering how I could monitor the CI runs for this userbenchmark?

Reply:
Raised #1821 addressing this comment